spec/system/admin/manage_order_detail_spec.rb

```diff
@@ -245,4 +246,38 @@
       expect(page).not_to have_button("Save")
     end
   end
+
+  describe "updating fulfilled date of a completed order" do
+    before do
+      order_detail.update!(state: :complete, fulfilled_at: 3.days.ago, order_status: OrderStatus.complete)
+      visit manage_facility_order_order_detail_path(facility, order, order_detail)
+    end
+
+    context "as global admin" do
+      let(:logged_in_user) { administrator }
+
+      before do
+        fill_in "order_detail[fulfilled_at]", with: DateTime.now.to_s
+        click_button "Save"
+        expect(page).to have_content("The order was successfully updated.")
+        click_link order_detail.to_s
+      end
+
+      it "can update the fulfilled at date" do
+        expect(page).to have_content SpecDateHelper.format_usa_date(Date.today)
+      end
+
+      it "logs fulfilled at date updates" do
+        expect(LogEvent).to be_exists(loggable: order_detail, event_type: :updated_fulfilled_at, user: logged_in_user)
+      end
+    end
+
+    context "as non-global admin" do
+      let(:logged_in_user) { director }
+
+      it "cannont update the fulfilled at date" do
+        expect(page).to have_content("Fulfilled Date").once
```

Comment:
I was thinking something a little more specific, like:
```suggestion
        expect(page).to have_no_selector("input", "order_detail[fulfilled_at]")
```